utils/docker-ce/files/dockerd.init

```diff
@@ -52,9 +52,20 @@ uciadd() {
 		exit 0
 	}
 
+	# Add docker bridge device
```

Comment:
I'm not super familiar with the network UCI, so why do we need a `bridge` type interface as well as a normal interface?
I would have thought that we could just `uci_quiet set network.docker.type="bridge"` on the one we already have.

Reply:
Recently there is a staged [pullrequest](https://github.com/openwrt/luci/pull/4307) from @jow- to create and manage devices in the LuCI.
So from the beginning an interface works on a device.
So I think we should add also the docker as a device to the uci because this is a bridge.
And on top of them we give them an ip.
This better reflects the reality on the system.

Reply:
Oh, `config device` is something that was added recently, thanks for the info